travis/shared_configs/anaconda-upload.yml

```diff
@@ -0,0 +1,34 @@
+jobs:
+  include:
+    - stage: deploy
+      name: "Anaconda Upload"
+      if: |
+        type != pull_request AND \
+        ((branch = master AND env(CONDA_UPLOAD_TOKEN_DEV) IS present) OR \
+        (branch = tag AND env(CONDA_UPLOAD_TOKEN_TAG) IS present))
+      python: 3.7
+      before_install:
+        - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+        - bash miniconda.sh -b -p $HOME/miniconda
+        - source $HOME/miniconda/etc/profile.d/conda.sh
+        - conda activate base
+        - hash -r
+        - conda config --set always_yes yes --set changeps1 no
+      install:
+        - conda install python=$TRAVIS_PYTHON_VERSION conda-build=3.12.0
```

Comment:
If we ever end up having CI builds that test OSX on Travis, we might consider decoupling the conda-installed Python version compared to `$TRAVIS_PYTHON_VERSION` - as at least the last time I checked, there are no OSX test containers for recent Python versions.

Also, why the pinning of `conda-build=3.12.0`?

Reply:
If that case ever comes around, a lot would have to change. For example, I don't think `python_tester.yml` would work at all.

The conda-build pinning was carried over from https://github.com/pcdshub/pcdsdevices/issues/399. I don't think any of us found a fix for this or even looked at it recently so I just left it in with the intention of unpinning when a solution is found.